tasmota/language/fr_FR.h

```diff
@@ -367,10 +367,10 @@
 #define D_MQTT_TOPIC "Topic MQTT"
 #define D_MQTT_GROUP_TOPIC "Groupe topic MQTT"
 #define D_MQTT_FULL_TOPIC "Topic complet MQTT"
-#define D_MQTT_NO_RETAIN "MQTT No Retain"
+#define D_MQTT_NO_RETAIN "MQTT No Retain" // Keep MQTT keyword
 #define D_MDNS_DISCOVERY "Découverte mDNS"
 #define D_MDNS_ADVERTISE "Annonce mDNS"
-#define D_ESP_CHIP_ID "ESP Chip Id"
+#define D_ESP_CHIP_ID "ESP Chip Id"  // Not better in french
```

Comment:
Id processeur

Reply:
Less informative IMO, I would keep ESP, What about "Id ESP" ?

Reply:
While such a translation may make good linguistic sense, my experience is that "force-translating" very technical terms gets too confusing, unless it is to use already well-established local terms, without loss of information, basically good as web search terms. And losing the "ESP" part may make it a useless search term. The point is that people not already knowing *exactly* what is meant would normally have to ask the almighty (Google or whatever) what it is.